RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -158,7 +158,7 @@
                            maxPtForLooperReconstruction = 0.0)    
 jetCoreRegionalStepBarrelTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilderIterativeDefault.clone(
     trajectoryFilter = dict(refToPSet_ = 'jetCoreRegionalStepBarrelTrajectoryFilter'),
-    maxCand = 50,
+    maxCand = 30,
```

Comment:
from a chat in the TRK mattermost thread, this change is not enough, another update is needed before this PR should be merged.